monai/apps/detection/transforms/dictionary.py

```diff
@@ -1307,6 +1308,7 @@ class RandRotateBox90d(RandRotate90d):
         spatial_axes: 2 int numbers, defines the plane to rotate with 2 spatial axes.
             Default: (0, 1), this is the first two axis in spatial dimensions.
         allow_missing_keys: don't raise exception if key is missing.
+        lazy:
```

Comment:
looks like this is work in progress?

Reply:
No just a docstring tidy up required. Initially I was adding in the lazy parameter but decided it would be fine without it, but didn't remove the docstring entry